client/my-sites/stats/stats-purchase/stats-purchase-tier-upgrade-slider.tsx

```diff
@@ -29,16 +39,34 @@ function useTranslatedStrings() {
 	};
 }
 
-function TierUpgradeSlider( { className }: TierUpgradeSliderProps ) {
+function TierUpgradeSlider( { className, priceTiers, currencyCode }: TierUpgradeSliderProps ) {
+	const translate = useTranslate();
+	const infoReferenceElement = useRef( null );
 	const componentClassNames = classNames( 'stats-tier-upgrade-slider', className );
+	const EXTENSION_THRESHOLD = 2; // in millions
```

Comment:
I didn't want to hard-code this, but adding a loop that finds the highest tier and multiplies the limit felt over the top as well. What do you think about this solution?